engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetViewportSizeOf.java

```diff
@@ -86,6 +84,9 @@ public void update() {
 
     @Override
     public String toString() { // TODO: used for logging purposes at the moment, investigate different methods
+        if (isTheDefaultInstance()) { // necessary since default instance is generated before setFrameBuffersManager
+            fbo = frameBuffersManager.getFBO(fboName);
+        }
         return String.format("%21s: %s(%sx%s)", this.getClass().getSimpleName(), fboName, fbo.width(), fbo.height());
```

Comment:
Didn't we discuss eliminating the fbo.width()/fbo.height() output from the state change log, keeping it only in the task's log instead?